### PR TITLE
README: Add "Chat" section

### PR DESCRIPTION
**Pull request by suutari-ai**
Promote the Gitter chat room with a badge.

### CODE REVIEW

**Comment by akx** [comment]
Malformed commit message. Also, "chat room".

**Comment by suutari-ai** [comment]
Fixed typo "root" -> "room" in the commit message. What do you mean by malformed?

**Comment by akx** [comment]
@suutari-ai: Title line not in imperative case.

> Readme: Promote the Gitter chat room with a badge.

would be enough imho :)

**Comment by suutari-ai** [comment]
It's a title (or headline). It should NOT end with a period, and it doesn't have to be a sentence. (Compare to newspaper headlines etc.) Most important function for the title is to easily identify bunch of commits (e.g. when reading the commit logs, shortlogs, "git gui blame" output, etc.). IMHO shortness matters there very much. That's why I prefer forms like "Foo: Bar" over "Add Bar to Foo" for the title and then use more words in the commit message body to explain it in more details.

File: README.md [hunk 1]
```diff
@@ -29,6 +29,13 @@ published under other AGPLv3-compatible licenses.  For these, please
 refer to VENDOR-LICENSES.md file in the source code tree or the licenses
 included within each package.
 
+## Chat
+
+We have a chat room for Shoop at Gitter.   Welcome to chat with us!
```
Comment (akx) [inline README.md]: > We have a Gitter chat room for Shoop. Come chat with us!

**Comment by suutari-ai** [comment]
Any better now?

**Pull request merged**